packages/swapkit/helpers/src/helpers/asset.ts

```diff
@@ -190,24 +193,29 @@ export const getAssetType = ({ chain, symbol }: { chain: Chain; symbol: string }
   if (symbol.includes("/")) return "Synth";
 
   switch (chain) {
+    case Chain.Arbitrum:
+      return [Chain.Ethereum, Chain.Arbitrum].includes(symbol as Chain) ? "Native" : "ARBITRUM";
+    case Chain.Optimism:
+      return [Chain.Ethereum, Chain.Optimism].includes(symbol as Chain) ? "Native" : "OPTIMISM";
+    case Chain.Base:
+      return [Chain.Ethereum, Chain.Base].includes(symbol as Chain) ? "Native" : "BASE";
+
     case Chain.Cosmos:
       return symbol === "ATOM" ? "Native" : Chain.Cosmos;
-    case Chain.Kujira:
-      return symbol === Chain.Kujira ? "Native" : Chain.Kujira;
     case Chain.BinanceSmartChain:
       return symbol === "BNB" ? "Native" : "BEP20";
-    case Chain.Ethereum:
-      return symbol === Chain.Ethereum ? "Native" : "ERC20";
+
+    case Chain.Kujira:
```

Comment:
You could probably combine some of those cases and return
return symbol === chain ? "Native" : chain;

Reply:
For some of them - sure - I will make "default" case like that

Reply:
thanks